src/functions/index.js

```diff
@@ -10,6 +11,14 @@ module.exports = function (fractal) {
         },
         url(str, obj) {
             return 'url://' + str + queryString.stringify(obj);
+        },
+        link(text, url = '', attributes = {}) {
```

Comment:
Happy to handle this in the same PR, but it should be a separate commit that is not a fixup and instead with proper commit message that explains what this change is about.